constraints.pro

```diff
@@ -281,11 +281,11 @@ gen_enforced_field(WorkspaceCwd, 'scripts.test', 'jest --reporters=jest-silent-r
   WorkspaceCwd \= '.'.
 
 % All non-root packages must have the same "test:clean" script.
-gen_enforced_field(WorkspaceCwd, 'scripts.test:clean', 'jest --clearCache') :-
+gen_enforced_field(WorkspaceCwd, 'scripts.test:only-clean', 'jest --clearCache && rm -rf coverage') :-
```

Comment:
Package-level `test:clean` did not have the same behavior as root-level `test:clean`. This highlights the differences.